flexmeasures/ui/templates/crud/account.html

```diff
@@ -1,183 +1,345 @@
-                                {% endif %}
-                            </td>
-                            <td>
-                                {{ asset.sensors | length }}
-                            </td>
-                            <td>
-                                <a href="/assets/{{ asset.id }}/status">
-                                    <button type="button" class="btn">Status</button>
-                                </a>
-                            </td>
-                            <td class="text-right">
-                            </td>
-                            <td class="d-none">
-                                /assets/{{ asset.id }}
-                            </td>
-                        </tr>
-                        {% endfor %}
-                    </tbody>
-                </table>
-            </div>
-            </div>
+                <!-- Name color field -->
+                <div class="form-group">
+                  <div class="col-md-3">
+                    <label for="name" class="control-label">Name</label>
+                    <input
+                      type="text"
+                      class="form-control"
+                      id="name"
+                      name="name"
+                      value="{{ account.name }}"
+                      required
+                    />
+                  </div>
+                </div>
+
+                <!-- Primary color field -->
+                <div class="form-group">
+                  <div class="col-md-3">
+                    <label for="primary_color" class="control-label"
+                      >Primary Color</label
+                    >
+                    <input
+                      type="text"
+                      class="form-control"
+                      id="primary_color"
+                      name="primary_color"
+                      value="{{ account.primary_color or primary_color }}"
+                    />
+                  </div>
+                </div>
+
+                <!-- Secondary color field -->
+                <div class="form-group">
+                  <div class="col-md-3">
+                    <label for="secondary_color" class="control-label"
+                      >Secondary Color</label
+                    >
+                    <input
+                      type="text"
+                      class="form-control"
+                      id="secondary_color"
+                      name="secondary_color"
+                      value="{{ account.secondary_color or secondary_color }}"
+                    />
+                  </div>
+                </div>
+
+                <!-- Loo URL field -->
+                <div class="form-group">
+                  <div class="col-md-3">
+                    <label for="logo_url" class="control-label">Logo URL</label>
+                    <input
+                      type="text"
+                      class="form-control"
+                      id="logo_url"
```

Comment:
Let's add `title="The logo to use when this platform is white-labelled and should show this account's branding."`